WORKSPACE

```diff
@@ -73,6 +73,20 @@ npm_translate_lock(
         "@aspect-test/c": "echo moo > cow.txt",
         "@aspect-test/c@2.0.2": "echo mooo >> cow.txt",
     },
+    data = [
```

Comment:
Can these just be globs?

Reply:
I don't think you can glob in a WORKSPACE.

Reply:
Correct. You can't glob in a repository rule